checker/src/main/java/org/checkerframework/checker/nullness/collection-object-parameters-may-be-null.astub

```diff
@@ -0,0 +1,152 @@
+// For documentation, see
+// https://checkerframework.org/manual/#collection-object-parameters-may-be-null .
+
+// The affected methods are:
+// containsKey(Object)
+// contains(Object)
+// containsValue(Object)
+// get(Object)
+// getOrDefault(Object, V)
+// indexOf(Object)
+// lastIndexOf(Object)
+// removeFirstOccurrence(Object)
+// removeLastOccurrence(Object )
+// remove(Object)
+// remove(Object, Object)
+
+package java.util;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public abstract class AbstractList<E> extends AbstractCollection<E> implements List<E> {
+    public int indexOf(@Nullable Object o);
+    public int lastIndexOf(@Nullable Object o);
+}
+
+public class ArrayList<E> extends AbstractList<E>
+        implements List<E>, RandomAccess, Cloneable, java.io.Serializable
+{
+    public boolean contains(@Nullable Object o);
+    public int indexOf(@Nullable Object o);
+    public int lastIndexOf(@Nullable Object o);
+    public boolean remove(@Nullable Object o);
+}
+
+public interface Collection<E> extends Iterable<E> {
+    boolean contains(@Nullable Object o);
+    boolean add(@Nullable E e);
+    boolean remove(@Nullable Object o);
+    boolean containsAll(Collection<? extends @Nullable Object> c);
+    boolean addAll(Collection<? extends @Nullable E> c);
+    boolean retainAll(Collection<? extends @Nullable Object> c);
+}
+
+public interface Comparator<T> {
+    int compare(@Nullable T o1, @Nullable T o2);
+
+}
+
+public interface Deque<E> extends Queue<E> {
+    boolean removeFirstOccurrence(@Nullable Object o);
+    boolean removeLastOccurrence(@Nullable Object o);
+    boolean remove(@Nullable Object o);
+    boolean contains(@Nullable Object o);
+}
+
+public interface List<E> extends Collection<E> {
+    boolean contains(@Nullable Object o);
+    boolean remove(@Nullable Object o);
+    int indexOf(@Nullable Object o);
+    int lastIndexOf(@Nullable Object o);
+}
+
+public interface Map<K, V> {
+    boolean containsKey(@Nullable Object key);
+    boolean containsValue(@Nullable Object value);
+    @Nullable V get(@Nullable Object key);
+    @Nullable V remove(@Nullable Object key);
+    default V getOrDefault(@Nullable Object key, V defaultValue);
+    default boolean remove(@Nullable Object key, @Nullable Object value);
+    }
+
+public class PriorityQueue<E> extends AbstractQueue<E>
```

Comment:
Do these stubs still need bounds, like `<E extends @NonNull Object>` in this case?

Reply:
They do not need them.  Because nothing is written there, the bounds from the annotated JDK are used.
It would do no harm to write them, and if you think it would be clearer to do so, I can do that.